src/pages/auth/component/AuthView.tsx

```diff
@@ -0,0 +1,43 @@
+import { defineComponent, onMounted } from 'vue'
+import { MaskView } from './MaskView'
+import './auth.less'
+import { useStore } from 'vuex'
+
+export const AuthView = defineComponent({
+  setup(props, { slots }) {
+    const $store = useStore()
+    const doClose = () => {
+      $store.commit('Auth/HIDE_VIEW')
+    }
+
+    onMounted(() => {
+      window.history.pushState(null, '123', location.href)
```

Comment:
What is the reason for this?

Reply:
push empty route, than modal can be closed by router.back()